src/segment-loader.js

```diff
@@ -2440,7 +2448,7 @@ export default class SegmentLoader extends videojs.EventTarget {
   updateTimingInfoEnd_(segmentInfo) {
     segmentInfo.timingInfo = segmentInfo.timingInfo || {};
     const useVideoTimingInfo =
-      this.loaderType_ === 'main' && this.startingMedia_.hasVideo;
+      this.loaderType_ === 'main' && this.startingMedia_ && this.startingMedia_.hasVideo;
```

Comment:
Since the check for startingMedia happens earlier, I think we can change this back

```suggestion
      this.loaderType_ === 'main' && this.startingMedia_.hasVideo;
```